sorc/checkout.sh

```diff
@@ -5,10 +5,7 @@ while getopts "o" option;
 do
  case $option in
   o)
-   echo "Received -o flag for optional checkout of GTG, will check out GTG with EMC_post"
-   checkout_gtg="YES"
-   checkout_wafs="YES"
-   gtg_git_args="--recursive"
+   echo "Received -o flag for operations"
```

Comment:
This is unnecessary now, but for the sake of continuity (and may be future operational specific options), I chose to keep it.

Reply:
Even if it was `YES`, it would do nothing. So.